components/tests.py

```diff
@@ -500,8 +502,38 @@ def setUpTestData(cls):
     def test_get_types_list(self):
         resp = self.client.get("/api/components/types/", format="json")
         self.assertEqual(resp.status_code, 200)
-        self.assertEqual(json.loads(resp.content)[0][0], "policy")
-        self.assertEqual(json.loads(resp.content)[1][0], "software")
+
+        content = resp.json()
+        # Data structure in response is [["type1"], ["type2"], ..]
+        flattened = sorted(value for item in content for value in item)
+        self.assertEqual(flattened, ["policy", "software"])
```

Comment:
Updated the test to be more values based rather than order based to prevent this test from failing due to order changes.

Reply:
:) I like this approach much better